### ISSUE TITLE
Console errors

### ISSUE THREAD

**Issue by albell** [winkerVSbecks/xvg#27]
This sounds like a great idea, but I can't seem to get it working at all. Can you advise?

![screen shot 2017-01-17 at 10 54 12 am](https://cloud.githubusercontent.com/assets/1142025/22035097/86bdbb10-dca3-11e6-9d68-0988b00444ee.png)

It might be nice to have a short looping demonstration video on the homepage.

**Comment by laurosollero** [comment]
It will work if your SVG is running inside an html, these errors are showing because you've tried to run it on the SVG file itself.

Yes, the extension could explain it better, test and not throw errors on console. But it seems to be working as expected for now.

**Comment by albell** [comment]
Understood.  For me, the ability to inspect and SVG file directly is an really desirable feature for a project like this. +1 for this feature. I can open a separate feature request if that's helpful.

**Comment by winkerVSbecks** [comment]
@albell I have implemented this already. Just waiting on another feature before I release an update.

**Comment by winkerVSbecks** [comment]
@albell pushed an update. This should now be fixed. (version 1.2.0)

**Comment by albell** [comment]
🎆  Excellent! Thank you.
